src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -53,18 +63,24 @@ OtpUser preCreateHook(OtpUser user, Request req) {
             Auth0Connection.ensureApiUserHasApiKey(req);
             user.applicationId = requestingUser.apiUser.id;
         }
-        if (Objects.nonNull(user.mobilityProfile)) {
-            user.mobilityProfile.updateMobilityMode();
-        }
+        preliminaryTasks(user);
         return super.preCreateHook(user, req);
     }
 
     @Override
     OtpUser preUpdateHook(OtpUser user, OtpUser preExistingUser, Request req) {
+        preliminaryTasks(user);
+        return super.preUpdateHook(user, preExistingUser, req);
+    }
+
+    /**
+     * Tasks to be carried out before creating or updating a user.
+     */
+    private void preliminaryTasks(OtpUser user) {
         if (Objects.nonNull(user.mobilityProfile)) {
             user.mobilityProfile.updateMobilityMode();
         }
-        return super.preUpdateHook(user, preExistingUser, req);
+        manageAcceptDependentEmail(user);
```

Comment:
I just realized there needs to be one more thing that needs to happen here. If a user removes a related user and PUTs the updated data, the `dependents` field in the `OtpUser` record for the removed user needs to be updated too.

Reply:
Addressed in: https://github.com/ibi-group/otp-middleware/pull/256/commits/4a00bfa58c2f747e1f645b4b2b0a4739ac136b6f